Sources/Orbit/Support/Layout/OptionalEmptyView.swift

```diff
@@ -0,0 +1,5 @@
+
+/// A type that represents views that can optionally result in `EmptyView`.
+protocol OptionalEmptyView {
```

Comment:
Wouldn't `PossiblyEmptyView` or `PotentiallyEmptyView` be more accurate? `Optional` has a meaning in Swift already...

Also, this is internal, so `///` doesn't make sense IMO.

Reply:
I was deciding between `PotentiallyEmptyView` and this one, will rename it. The comment is there mostly for orbit devs. But the better name may make it less necessary.